### PR TITLE
Prepare repo for automatically updating az_quickstart on new release.

### PR DESCRIPTION
**Pull request by trackleft**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
+FROM node:14.16.0-buster-slim
```
Comment (joeparsons) [inline Dockerfile:1]: Would it be worth switching to the active LTS node release (16)?

Comment (trackleft) [reply Dockerfile:1]: yes

**Comment by joeparsons** [comment]
I think we need to either link this to a new issue or at least add a description to this PR.

**Comment by joeparsons** [comment]
We recently made some changes to the analogous GH Actions workflows in the Arizona Bootstrap repo that we may need to apply here.

**Comment by trackleft** [comment]
Superceded by #35

**Pull request closed without merging**